backstop/tests/color.html

```diff
@@ -0,0 +1,255 @@
+        </ul>
+      </div>
+      <div class="demo-background-4">
+        <h2 class="iui-text-title iui-text-spacing">Background 4</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+          <li class="iui-text-block iui-text-muted">Muted</li>
+        </ul>
+      </div>
+      <div class="demo-background-5">
+        <h2 class="iui-text-title iui-text-spacing">Background 5</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+          <li class="iui-text-block iui-text-muted">Muted</li>
+        </ul>
+      </div>
+
+      <div class="demo-status-primary">
+        <h2 class="iui-text-title iui-text-spacing">Background primary</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+        </ul>
+      </div>
+      <div class="demo-status-positive">
+        <h2 class="iui-text-title iui-text-spacing">Background positive</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+        </ul>
+      </div>
+      <div class="demo-status-warning">
+        <h2 class="iui-text-title iui-text-spacing">Background warning</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+        </ul>
+      </div>
+      <div class="demo-status-negative">
+        <h2 class="iui-text-title iui-text-spacing">Background negative</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+        </ul>
+      </div>
+
+      <div class="demo-status-overlay-primary">
+        <h2 class="iui-text-title iui-text-spacing">Overlay primary</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+          <li class="iui-text-block iui-text-muted">Muted</li>
+          <li class="demo-text-primary">Primary</li>
+          <li class="demo-text-primary-muted">Primary muted</li>
+        </ul>
+      </div>
+      <div class="demo-status-overlay-positive">
+        <h2 class="iui-text-title iui-text-spacing">Overlay positive</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+          <li class="iui-text-block iui-text-muted">Muted</li>
+          <li class="demo-text-positive">Positive</li>
+          <li class="demo-text-positive-muted">Positive muted</li>
+        </ul>
+      </div>
+      <div class="demo-status-overlay-warning">
+        <h2 class="iui-text-title iui-text-spacing">Overlay warning</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+          <li class="iui-text-block iui-text-muted">Muted</li>
+          <li class="demo-text-warning">Warning</li>
+          <li class="demo-text-warning-muted">Warning muted</li>
+        </ul>
+      </div>
+      <div class="demo-status-overlay-negative">
+        <h2 class="iui-text-title iui-text-spacing">Overlay negative</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+          <li class="iui-text-block iui-text-muted">Muted</li>
+          <li class="demo-text-negative">Negative</li>
+          <li class="demo-text-negative-muted">Negative muted</li>
```

Comment:
Positive / Warning / Negative muted text on overlay background are not present anywhere in our UI.  However primary muted text on overlay is present in the header buttons.